src/BloomExe/Publish/Epub/EpubMaker.cs

```diff
@@ -2208,10 +2208,18 @@ private void RemoveRegularStylesheets (HtmlDom pageDom)
 					continue;
 				if (href != null && Path.GetFileName(href.Value) == "defaultLangStyles.css")
 					continue;
-				// BL-9844 Let's keep the Kyrgyzstan xmatter stylesheet
-				// (just for 4.9/5.0; we will keep ePub-specific css in 5.1).
+				// BL-9844, BL-10080 We need some special style rules for Kyrgyzstan2020
+				// Xmatter even in epubs, but including the whole standard stylesheet
+				// doesn't work well in many readers, which don't handle various
+				// sophisticated techniques it uses like css variables and, in some readers,
+				// flexbox. So we have a custom Kyrgyzstan2020 style sheet for epubs.
+				// In 5.1 there will be a more general approach to supporting custom
+				// xmatter stylesheets for epubs.
 				if (href != null && Path.GetFileName(href.Value).StartsWith("Kyrgyzstan2020"))
+				{
+					link.SetAttribute("href", "Kyrgyzstan2020-Xmatter-epub.css");
 					continue;
```

Comment:
If I'm reading this correctly, it will still include Kyrgyzstan2020-Xmatter.css which we don't want, right?